radixdlt/babylon-wallet-android
# [ADB-2674] Resolve dapps associated with pools

## Description
This PR resolves dApp related to pools and displays them under "Contributing/Redeeming to pools" section. If dapp exist but it is not verified (dapp does not have link to pool component) - it is displayed as unknown pool.

## How to test

1. Contribute to pool or reedem pool units
2. Validate that Pools section is populated - if you use sandbox dapp, you will see "1 pool component"
3. I'm not sure how to test this for a pool that has a reference to dapp, so I tested it by mocking this scenario.

## PR submission checklist
- [ ] I have tested that associated dapps are rendered in pool tx preview
